ghost/i18n/locales/lv/newsletter.json

```diff
@@ -0,0 +1,24 @@
+{
+    "By {authors}": "Autors: {authors}",
+    "Comment": "komentēt",
+    "complimentary": "bezmaksas",
+    "Email": "E-pasts",
+    "free": "bezmaksas",
+    "Keep reading": "Turpiniet lasīt",
+    "Less like this": "Mazāk kā šis",
+    "Manage subscription": "Pārvaldīt abonementu",
+    "Member since": "Dalībnieks kopš",
+    "More like this": "Vairāk kā šis",
+    "Name": "Vārds",
+    "paid": "samaksāts",
+    "Subscription details": "Informācija par abonementu",
+    "trialing": "tiesāšana",

```

Comment:
From ChatGPT: "Tiesāšana" translates to "legal trial" (as in a court). For subscription trials, the appropriate translation might be "izmēģinājuma periods" or "izmēģinājums".